editor/src/components/canvas/controls/guideline-helpers.ts

```diff
@@ -17,9 +17,31 @@ import {
 import { ElementInstanceMetadataMap } from '../../../core/shared/element-template'
 import { MetadataUtils } from '../../../core/model/element-metadata-utils'
 import { EdgePosition } from '../canvas-types'
-import { pluck } from '../../../core/shared/array-utils'
 import { defaultIfNull } from '../../../core/shared/optional-utils'
 
+export function collectParentsAndSiblings(
```

Comment:
This looks like something which could be in MetadataUtils, it is independent from the guidelines

Reply:
moved, thanks!